WordPress/Classes/ViewRelated/Comments/WPStyleGuide+Comments.swift

```diff
@@ -66,7 +65,7 @@ extension WPStyleGuide
         
         public static func timestampImage(isApproved approved: Bool) -> UIImage {
             let timestampImage = UIImage(named: "reader-postaction-time")!
-            return approved ? timestampImage : timestampImage.imageTintedWithColor(WPStyleGuide.alertYellowDark())
+            return approved ? timestampImage : timestampImage.imageWithRenderingMode(UIImageRenderingMode.AlwaysTemplate)
```

Comment:
No need to specify `UIImageRenderingMode.`, you can just do:

`timestampImage.imageWithRenderingMode(.AlwaysTemplate)`